src/drivers/neon.ts

```diff
@@ -0,0 +1,99 @@
+import { createError, createRequiredError, defineDriver } from "./utils";
+import { neon, NeonQueryFunction, neonConfig } from "@neondatabase/serverless";
+
+export interface NeonDriverOptions {
```

Comment:
Could this object accept a `driverConfig` of type `NeonConfig` from the `@neondatabase/serverless`. This would allow users to supply other configuration options for the driver, or is that a little too leaky?

Reply:
Actually, nevermind. Users could just import `@neondatabase/serverless` and directly modify the `neonConfig`, so does the `fetchConnectionCache ` config need to be handled by this at all?

Reply:
Left it in to be consistent with the use in blog reference and docs. Also, added a short section highlight the additional advanced config, if needed.